### PR TITLE
Add transfers from extrinsic for failed things

### PR DESCRIPTION
**Pull request by hobbydev71**

### CODE REVIEW

File: src/mappings/mappingSystemTokenHandlers.ts [hunk 1]
```diff
@@ -22,4 +22,18 @@ export async function systemTokenTransferEvent(event: SubstrateEvent): Promise<v
...
     await record.save();
+}
+
+export async function systemTokenTransferCall(extrinsic: SubstrateExtrinsic): Promise<void> {
+    const blockNumber = (extrinsic.block.block.header.number as Compact<BlockNumber>).toNumber();
+    let record = new SystemTokenTransfer(blockNumber.toString() + '-' + extrinsic.idx.toString());
+    record.from = extrinsic?.extrinsic?.signer?.toString();
+    record.to = extrinsic.extrinsic.args[0].toString();
+    record.txHash = extrinsic.extrinsic.hash.toString();
+    record.amount = (extrinsic.extrinsic.args[1] as Balance).toBigInt();
```
Comment (secretfader) [inline src/mappings/mappingSystemTokenHandlers.ts:33]: Do we guarantee the `amount` field is always the second indexed item? If so, seems good to me, but this stood out when reading.

**Review by secretfader — DISMISSED** [review]
One question but otherwise looks good to me.

**Review by Douglasacost — DISMISSED** [review]
LGTM!

File: project.yaml [hunk 2]
```diff
@@ -30,11 +29,13 @@ dataSources:
...
-          kind: substrate/EventHandler
+        - handler: systemTokenTransferCall
+          kind: substrate/CallHandler
           filter:
-            module: vesting
-            event: VestingScheduleAdded
+            moudle: balances
+            method: transfer
+        - handler: handleVestingSchedule
+          kind: substrate/CallHandler
```
Comment (aliXsed) [inline project.yaml:38]: Why we are changing from indexing events to indexing call?

**Comment by Douglasacost** [comment]
Rejected as we have breaking changes.

**Pull request closed without merging**